src/praisonai/praisonai/gateway/kanban_dispatcher.py

```
@@ -0,0 +1,389 @@
+                store = self._get_kanban_store()
+                if store:
+                    self._cleanup_completed_tasks(store)
+                await asyncio.sleep(1)
+            
+            # Force terminate remaining tasks
+            for task_id, process in self.running_tasks.items():
+                try:
+                    logger.warning(f"Force terminating task {task_id}")
+                    process.terminate()
+                    process.wait(timeout=5)
+                except:
```

Ruff (0.15.14)

[error] Do not use bare `except`

(E722)